custom-words.txt

```diff
@@ -2740,3 +2740,5 @@ KubeProxyConfig
 IPVS
 TCPFIN
 taginheritance
+integrationruntimes
```

Comment:
Don't add this term. Use proper camelCase, PascalCase or English: "integrationRuntimes", "IntegrationRuntimes", or "integration runtimes"